### ISSUE TITLE
Leftover `uid` styles during `--watch`/`--serve`

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy-plugin-webc#17]
```html
<style @html="`#${uid} { --position: ${value}%; }`"></style>
```

Looks like those are sticking around even when the `uid` changes between builds

```css
#webc-q7gY8 { --position: 60%; }#webc-ehXEJ { --position: 60%; }#webc-CYrCA { --position: 60%; }#webc-lWwYc { --position: 60%; }#webc-P_AL3 { --position: 60%; }
```

Marginally related to #15

**Comment by zachleat** [comment]
We could make the `uid` consistent between builds but I think it’d be more prudent to clear the asset aggregation.

**Comment by zachleat** [comment]
This fix will ship with WebC v0.6.3

**Comment by zachleat** [comment]
Moving this one to 0.7.0 instead!
